examples/pages/semantic-ui/index.js

```diff
@@ -160,7 +160,6 @@ function SemanticUIAutocomplete() {
                 <XIcon />
               </ControllerButton>
               : <ControllerButton
-                onClick={toggleMenu}
                 aria-label={isOpen ? 'close menu' : 'open menu'}
```

Comment:
Can we remove this? The built in bottom should now handle all a11y for us right?